app/config/config.yml

```diff
@@ -9,6 +9,7 @@ parameters:
     api_connect_timeout_slow: 3
     api_timeout_slow: 10
     honeypot_field: email_address
+    instance: default
```

Comment:
Need a better name, will invalidate caches currently.

Reply:
Alternative is to ignore all this and update `AppKernel::getCacheDir()` (since tests put everything in the filesystem). Might then work with Behat/Mink too (they'd need to have a cookie or something for JS tests).

Reply:
In a Selenium context (and without sessions) the only reliable token is a cookie, as other headers can't usually be modified and query string parameters are lost on redirects and difficult to pass everywhere.